docs/development.md

```diff
@@ -0,0 +1,31 @@
+# Development Guide
+
+This document explains how to develop cluster-api-provide-openstack.
+
+## Using your own openstack-cluster-api-controller image for testing
+
+1. Building your own openstack-cluster-api-controller image
+1. Using your own openstack-cluster-api-controller image
+
+### Building your own openstack-cluster-api-controller image
+
+Set environment variables which is used in Makefile.
```

Comment:
do we really need set all those of some of them are optional (has default value? )

Reply:
VERSION is not mandatory, but specifying image tag is the best practice.

Reply:
Please note which env vars are mandatory/optional, and also add an example of each env var.

Reply:
I will add Mandatory and Example fields.